src/shared/components/query/studyList/StudyList.tsx

```diff
@@ -216,94 +228,117 @@ export default class StudyList extends QueryStoreComponent<IStudyListProps, {}>
-							>View study summary</div>
-						}
-						children={
-							<span onClick={()=>openStudySummaryFormSubmit(study.studyId)}
-							  className={ classNames(styles.summaryIcon, 'ci ci-pie-chart')}></span>
-						}
-					/>
-				)}
-			</span>
-		);
+    
+            return (
+                <span className={styles.StudyLinks}>
+                    {links.map((link, i) => {
+                        let content = (
+                            <FontAwesome
+                                key={i}
+                                name={link.icon}
+                                className={classNames({
+                                    [styles.icon]: true,
+                                    [styles.iconWithTooltip]: !!link.tooltip,
+                                    [styles.trashIcon]: (link.icon === "trash")
+                                })}
+                            />
+                        );
+    
+                        if (link.onClick) {
+                            let anchorProps:any = {
+                                key: i
+                            };
+                            if (typeof link.onClick === "string") {
+                                anchorProps.href = link.onClick;
+                                anchorProps.target = "_blank";
+                            } else {
+                                anchorProps.onClick = link.onClick;
+                            }
+                            content = (
+                                <a {...anchorProps}>
+                                    {content}
+                                </a>
+                            );
+                        }
+    
+                        if (link.tooltip)
+                        {
+                            let overlay = (
+                                <div className={styles.tooltip} dangerouslySetInnerHTML={{__html: link.tooltip}}/>
+                            );
+                            content = (
+                                <DefaultTooltip
+                                    key={i}
+                                    mouseEnterDelay={0}
+                                    placement="top"
+                                    overlay={overlay}
+                                    children={content}
+                                />
+                            );
+                        }
+    
+                        return content;
+                    })}
+                    {study.studyId && (
+                        <DefaultTooltip
+                            mouseEnterDelay={0}
+                            placement="top"
+                            overlay={
+                                <div className={styles.tooltip}
+                                >View study summary</div>
+                            }
+                            children={
+                                <span onClick={()=>openStudySummaryFormSubmit(study.studyId)}
```

Comment:
maybe just put this inside tag instead of assigning to children property

Reply:
true, I just changes the parent logic and ported everything what was present. I'll try to update this